FileBrowser/FileListViewController.swift

```diff
@@ -102,7 +103,9 @@ class FileListViewController: UIViewController {
     }
     
     @objc func dismiss(button: UIBarButtonItem = UIBarButtonItem()) {
-        self.dismiss(animated: true, completion: nil)
+        self.dismiss(animated: true, completion: {
+            self.willClose?()
```

Comment:
Should be didClose().

Reply:
Done.  This PR is closed, but there is an identical one that's open (I wasn't expecting a PR on my fork to go into your repo, though I don't mind that it did).